transmart-notifications/build.gradle

```diff
@@ -1,17 +1,27 @@
 dependencies {
+    //Grails 3 dependencies
     compile "org.springframework.boot:spring-boot-starter-logging"
     compile "org.springframework.boot:spring-boot-autoconfigure"
     compile "org.grails:grails-core"
-    compile "org.grails:grails-dependencies"
+    compile "org.springframework.boot:spring-boot-starter-actuator"
```

Comment:
Are these all needed?

Reply:
these are the default (auto-generated) dependencies for grails rest-api profiled plugin (I removed only the hibernate ones), but you're right, probably some of them are not needed, I will do some more cleanup